src/JBFundAccessLimits.sol

```diff
@@ -338,7 +338,7 @@ contract JBFundAccessLimits is JBControlled, IJBFundAccessLimits {
                 // Set the surplus allowance if there is one.
                 if (surplusAllowance.amount > 0) {
                     _packedSurplusAllowancesDataOf[projectId][rulesetId][fundAccessLimitGroup[i].terminal][fundAccessLimitGroup[i]
-                        .token].push(surplusAllowance.amount | (surplusAllowance.currency << 224));
+                        .token].push(uint256(surplusAllowance.amount) | (uint256(surplusAllowance.currency) << 224));
```

Comment:
Since the amount & currency packing is performed the same for surplus and payouts, we could declare this under an internal utility function. But also contract size isnt a problem here.

Reply:
true. im cool with it as is.